src/main/java/org/cryptomator/ui/updatereminder/UpdateReminderComponent.java

```diff
@@ -23,7 +24,9 @@ public interface UpdateReminderComponent {
 	Settings settings();
 
 	default void checkAndShowUpdateReminderWindow() {
-		if (LocalDate.parse(settings().lastUpdateCheck.get()).isBefore(LocalDate.now().minusDays(14)) && !settings().checkForUpdates.getValue()) {
+		var now = Instant.now();
+		if (settings().lastUpdateReminder.get().isBefore(now.minus(Duration.ofDays(14))) && !settings().checkForUpdates.getValue()) {
+			settings().lastUpdateReminder.set(now);
```

Comment:
This changes the logic quite a bit.

Before, we reminded, if the last *check* is 14 days ago.

Now, we remind, if the last *reminder* was 14 days ago - even if we just checked yesterday. I think this is too chatty.